rosetta/services/account.go

```diff
@@ -104,6 +155,39 @@ func newAccountIdentifier(
 	}, nil
 }
 
+// newSubAccountIdentifier ..
```

Comment:
Just remove comments when you don't want to write comments in private functions. ;)